apps/emqx_bridge/src/emqx_bridge_v2.erl

```diff
@@ -1052,6 +1046,18 @@ id(BridgeType, BridgeName, ConnectorName) ->
 source_id(BridgeType, BridgeName, ConnectorName) ->
     id_with_root_name(?ROOT_KEY_SOURCES, BridgeType, BridgeName, ConnectorName).
 
+get_resource_ids(ConfRootKey, Type, Name) ->
+    try
+        ChannelResId = id_with_root_name(ConfRootKey, Type, Name),
```

Comment:
cannot seem to find a `throw` down this call

Reply:
https://github.com/emqx/emqx/blob/23f18a3daccf9975664c4a105d56e0332361db79/apps/emqx_bridge/src/emqx_bridge_v2.erl#L1060-L1067